test/audienceCosineSimMultipleVectors.test.ts

```diff
@@ -0,0 +1,133 @@
+import { AudienceDefinition, QueryFilterComparisonType } from '../types';
+import { edkt } from '../src';
+
+/* TODO Mock localStorage and merge single and
```

Comment:
What do you mean here by mock localStorage?

Reply:
In the current setting, localStorage as well as ViewStore, MatchedAudienceStore and CachedAudienceStore, are being used raw and the asserting are being made over their contents on each test suite.
This setting is weak as it cannot assure that their methods are being called with the proper expected inputs.
It can happens, for an example, that the actual content of one of these was inserted on some unrelated test suite and is being
used spuriously on an assertion.
This setting also does not work well on a property based setup where the methods should get called a lot of times with random inputs.

I'll try to improve on this tomorrow when working on the property based implementations.

Reply:
gotha! sounds good.